docker/entrypoint-unit-tests-devDocker.sh

```diff
@@ -54,10 +54,11 @@ EOF
 echo "Unit Tests"
 echo "------------------------------------------------------------"
 
-python3 manage.py test unittests -v 3 --keepdb --no-input --failfast --shuffle --parallel --exclude-tag="non-parallel" || {
+# Removing parallel and shufful for now to maintain stability
+python3 manage.py test unittests -v 3 --keepdb --no-input --exclude-tag="non-parallel" || {
```

Comment:
Should we resume running these "non-parallel" tests again, at least while we have `--parallel` disabled?

```suggestion
python3 manage.py test unittests -v 3 --keepdb --no-input || {
```